.github/workflows/libkineto_cuda.yml

```
@@ -54,15 +58,16 @@ jobs:
             --detach \
             --shm-size=32gb \
             -v "${PWD}/kineto:/kineto" \
+            --gpus all \
             -w / \
             "${{ env.DOCKER_IMAGE }}" \
             tail -f /dev/null
           )
           echo "Container name: ${container_name}"
           docker exec -t -w "/" "${container_name}" bash -c "sudo chown -R runner /kineto; sudo chgrp -R runner /kineto"
           docker exec -t -w "/kineto" "${container_name}" bash -c "set -e; mkdir build_static; mkdir build_shared"
-          docker exec -t -w "/kineto/build_static" "${container_name}" bash -c "cmake -DKINETO_LIBRARY_TYPE=static ../libkineto/; make -j"
-          docker exec -t -w "/kineto/build_shared" "${container_name}" bash -c "cmake -DKINETO_LIBRARY_TYPE=shared ../libkineto/; make -j"
+          docker exec -t -w "/kineto/build_static" "${container_name}" bash -c "cmake -DCMAKE_CUDA_COMPILER:PATH=/usr/local/cuda/bin/nvcc -DKINETO_LIBRARY_TYPE=static ../libkineto/; make -j"
```

Can you try change this to the following?
```suggestion
          docker exec -t -w "/kineto/build_static" "${container_name}" bash -c ". /workspace/setup_instance.sh; cmake -DCMAKE_CUDA_COMPILER:PATH=/usr/local/cuda/bin/nvcc -DKINETO_LIBRARY_TYPE=static ../libkineto/; make -j"
```

---

Thanks! This change was landed in https://github.com/pytorch/kineto/pull/869